bindings/ldk_node.udl

```diff
@@ -446,6 +450,10 @@ dictionary PaymentDetails {
 	u64 latest_update_timestamp;
 };
 
+dictionary PaymentPreimage {
+	bytes inner;
```

Comment:
I wonder about what would be the most appropriate description for `[u8;N]`: `bytes` vs `sequence<u8>`?